.github/workflows/python-package.yml

```diff
@@ -27,9 +27,9 @@ jobs:
     - name: set environment for branch
       run: |
           if [[ "${{github.base_ref}}" == "master" || "${{github.ref}}" == "refs/heads/master" ]]; then
-            echo "::set-env name=LABELBOX_TEST_ENVIRON::prod"
```

Comment:
seems like `set-env` is deprecated? https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

Reply:
Yes, this is the point of this PR, removing `set-env` in favor of concatenating to an env-file, as linked on the page you're referencing: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable